datagen/ScaleTest.md

```diff
@@ -18,17 +18,42 @@ It will be submitted as the main class to Spark to run the data generation.
 ### User interface
 
 The input arguments for this tool is described as below:
+
+```bash
+Usage: DataGenEntry [options] <scale factor> <complexity> <format> <output directory>
+
+  <scale factor>        scale factor for data size
+  <complexity>          complexity level for processing
+  <format>              output format for the data
+  <output directory>    output directory for data generated
+  -t, --tables <value>  tables to generate. If not specified, all tables will be generated
+  -d, --seed <value>    seed used to generate random data columns. default is 41 if not specified
+  --overwrite           Flag argument. Whether to overwrite the existing data in the path.
+
 ```
-  -s, --scale_factor <value> scale factor for data size
-  -c, --complexity <value>   complexity level for processing
-  -t, --tables <value>       tables to generate
-  -o, --output_dir <value>   output directory
-  -f, --format <value>       output format
-  -v, --version <value>      version
-  -d, --seed <value>         seed used to generate random data columns
 
+The data generation tool can be used just like a normal Spark application, user
+can submit it to Spark master with essential parameters:
+
+```bash
+$SPARK_HOME/bin/spark-submit \
+--master spark://<SPARK_MASTER>:7077 \
+--conf spark.driver.memory=10G \
+--conf spark.executor.memory=32G \
+--conf spark.sql.parquet.int96RebaseModeInWrite=CORRECTED \
+--conf spark.sql.parquet.datetimeRebaseModeInWrite=CORRECTED \
+--class com.nvidia.rapids.tests.scaletest.ScaleTestDataGen \ # the main class
+--jars $SPARK_HOME/examples/jars/scopt_2.12-3.7.1.jar \ # one dependency jar just shipped with Spark under $SPARK_HOME
+./target/datagen_2.12-23.10.0-SNAPSHOT-spark332.jar \
```

Comment:
I thought that jars needed to be comma separated.

Reply:
You mean putting the applcation jar into `--jars`? In this case, the `./target/datagen_2.12-23.10.0-SNAPSHOT-spark332.jar` is actually the `application jar` who contains both the application entry class and also the data gen library code.  So the pattern here is
```
--jars scopt.jar \
<application-jar>
```
where application jar is a must and an independent argument for `spark-submit`.

Take our plugin jar for example if we want to use GPU generation, the command becomes 
```
--jars scopt.jar, spark-rapids.jar \
<application-jar>
```